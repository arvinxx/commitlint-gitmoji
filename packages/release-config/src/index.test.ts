import defaultConfig from './index';

test('get default config', () => {
  expect(defaultConfig).toMatchSnapshot();
});
