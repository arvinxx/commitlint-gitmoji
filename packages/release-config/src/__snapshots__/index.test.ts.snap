// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get default config 1`] = `
{
  "branches": [
    "master",
    {
      "channel": "rc",
      "name": "rc-*",
      "prerelease": "rc",
    },
    {
      "name": "rc",
      "prerelease": true,
    },
    {
      "channel": "alpha",
      "name": "alpha",
      "prerelease": "alpha",
    },
    "beta",
  ],
  "plugins": [
    [
      "@semantic-release/commit-analyzer",
      {
        "config": "conventional-changelog-gitmoji-config",
        "releaseRules": [
          {
            "release": "patch",
            "type": "style",
          },
          {
            "release": "patch",
            "type": "build",
          },
        ],
      },
    ],
    [
      "@semantic-release/release-notes-generator",
      {
        "config": "conventional-changelog-gitmoji-config",
      },
    ],
    [
      "@semantic-release/changelog",
      {
        "changelogFile": "CHANGELOG.md",
        "changelogTitle": "# Changelog",
      },
    ],
    "@semantic-release/npm",
    "@semantic-release/github",
    [
      "@semantic-release/git",
      {
        "assets": [
          "CHANGELOG.md",
          "package.json",
        ],
        "message": ":bookmark: chore(release): \${nextRelease.gitTag} [skip ci] 

\${nextRelease.notes}",
      },
    ],
  ],
}
`;
